amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/Scheduler.java

```diff
@@ -180,7 +180,7 @@ public class Scheduler implements Runnable {
     private final Object lock = new Object();
 
     private final Stopwatch streamSyncWatch = Stopwatch.createUnstarted();
-    private boolean leasesSyncedOnAppInit = false;
+    private boolean shouldSyncLeases = true;
```

Comment:
Unfortunately, this is not a backwards-compatible change since `Scheduler` has exposed its member variables via accessors. I'll reach out to you internally so we can discuss release strategy.

Reply:
Created separate non-assessable variable and kept the current logic untouched